packages/flutter/lib/src/gestures/arena.dart

```diff
@@ -266,7 +266,7 @@ class GestureArenaManager {
   bool _debugLogDiagnostic(int pointer, String message, [ _GestureArena state ]) {
     assert(() {
       if (debugPrintGestureArenaDiagnostics) {
-        final int count = state != null ? state.members.length : null;
+        final int count = state?.members?.length;
```

Comment:
Isn't this more expensive now?  Before we only checked if `state` was null, now we have to check both `state` and `state.members` (and presumably, there's some contract saying `members` is never null).

Reply:
this is in an assert, performance doesn't matter